packages/runtime/src/runtimes/react/controls/group.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+  GroupControl,
+  GroupDefinition,
+  type DataType,
+  type ResolvedValueType,
+} from '@makeswift/controls'
+
+import { ControlValue } from './control'
+
+type GroupControlValueProps = {
+  definition: GroupDefinition
+  data: DataType<GroupDefinition> | undefined
+  children(value: ResolvedValueType<GroupDefinition>): JSX.Element
+  control: GroupControl
+}
+
+export function GroupControlValue({ definition, data, children, control }: GroupControlValueProps) {
```

Comment:
Renamed [shape-v2.tsx](https://github.com/makeswift/makeswift/pull/909/files?diff=split&w=1#diff-0fdfe8a7b77d50d78ccd537ded6178fbc58d25bf9c9998236fd2d77ba61caf8d). `git` doesn't recognize this as a rename due to a simplified implementation.